### PR TITLE
Standardize and add granular, descriptive page titles

### LINKED ISSUE #866: Pages should have actual titles.
Right now every page basically has the same title. 
- Since every page is titled the same, when I hold the back button on Chrome to figure out my path through OK, it's impossible to pick the previous page I want. So, I don't know whether I'm jumping back to a particular assignment or the assignments overview page, for example.

Tasks involved: 
- Look through all of the templates and change the title block. 
- Some tests will break, change the test to match the new page title.

### PR DESCRIPTION
**Pull request by colinschoen**
Fixes #866 

Signed-off-by: Colin Schoen <cschoen@berkeley.edu>

### CODE REVIEW

**Comment by colinschoen** [comment]
PTAL

File: server/templates/staff/student/overview.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% extends "staff/base.html" %}
 
-{% block title %}Ok - {{ current_course.display_name_with_semester }}{% endblock %}
+{% block title %} {{ student.email }} - {{ current_course.display_name_with_semester }}{% endblock %}
```
Comment (Sumukh) [inline server/templates/staff/student/overview.html]: student.identifier ?

Comment (colinschoen) [inline server/templates/staff/student/overview.html]: Done.

File: server/templates/staff/jobs/moss.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {% extends "staff/base.html" %}
 {% import "staff/_formhelpers.html" as forms %}
 
-{% block title %} Ok - Start Moss Job {% endblock %}
+{% block title %} Moss -  {{ current_course.display_name_with_semester }} {% endblock %}
```
Comment (colinschoen) [inline server/templates/staff/jobs/moss.html]: Done.

File: server/templates/staff/student/submit.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {% extends "staff/base.html" %}
 {% import 'staff/_formhelpers.html' as forms with context %}
 
-{% block title %} Ok - New Submission for {{ student.identifier }}{% endblock %}
+{% block title %} Submit Assignment - {{ current_course.display_name_with_semester }}{% endblock %}
```
Comment (Sumukh) [inline server/templates/staff/student/submit.html]: instead of submit for {{ course.display_name }} - we might want to keep the student.identifier here.

Comment (colinschoen) [inline server/templates/staff/student/submit.html]: Done.

**Comment by colinschoen** [comment]
Thanks for reviewing @Sumukh

**Pull request merged**